src/oscar/apps/catalogue/abstract_models.py

```diff
@@ -367,8 +369,10 @@ class AbstractProduct(models.Model):
     # Title is mandatory for canonical products but optional for child products
     title = models.CharField(pgettext_lazy('Product title', 'Title'),
                              max_length=255, blank=True)
-    slug = models.SlugField(_('Slug'), max_length=255, unique=False)
+    slug = models.SlugField(_('Slug'), max_length=255, unique=False, blank=True)
```

Comment:
Why has this field changed? This PR should not be altering slug handling at all?

Reply:
Without it, the dashboard's `ProductForm` tests would fail since `slug` wasn't part of the form's fields before and it's a required field in the model class which would make it required in the form without the change.